src/Test.UnitTests.Sarif/Visitors/InsertOptionalDataVisitorTests.cs

```diff
@@ -246,6 +253,13 @@ public void InsertOptionalDataVisitor_ContextRegionSnippets_DoesNotFail_TopLevel
                 OptionallyEmittedData.ContextRegionSnippets);
         }
 
+        [Fact]
+        public void InsertOptionalDataVisitor_PartialFingerprint_DoesNotFail_TopLevelOriginalUriBaseIdUriMissing()
```

Comment:
>InsertOptionalDataVisitor_PartialFingerprint_DoesNotFail_TopLevelOriginalUriBaseIdUriMissing [](http://example.com/codeflow?start=20&length=92)

seems this is same as above test case. please lets know if its not. we try to not add test SARIF files if not necessary, to avoid increasing tests run time since these tests need disk I/O.

Reply:
Yes you are right, it is almost the same.